offscreen_effect_player/offscreen_effect_player.cpp

```diff
@@ -43,9 +44,10 @@ namespace bnb::oep
     /* offscreen_effect_player::~offscreen_effect_player */
     offscreen_effect_player::~offscreen_effect_player()
     {
-        m_ep->surface_destroyed();
-        // Deinitialize offscreen render target, should be performed on render thread.
+        // Switches playback state to inactive state and deinitializes offscreen render target.
+        // Must be performed on render thread.
         auto task = [this]() {
+            m_ep->surface_destroyed();
```

Comment:
Why is the surface_destroyed should be called in the render thread?

Reply:
Because this is states in the description of the bnb::interfaces::effect_player::surface_destroyed(). 
There were freezes of the app (during exit) until this change.

Reply:
OK